build/squire-raw.js

```diff
@@ -2254,17 +2254,33 @@ var onDrop = function ( event ) {
     while ( l-- ) {
         switch ( types[l] ) {
         case 'text/plain':
+        case 'Text': // IE Specific
             hasPlain = true;
             break;
         case 'text/html':
             hasHTML = true;
             break;
         default:
-            return;
+            break;
         }
     }
+
     if ( hasHTML || hasPlain ) {
+        var range = this.getSelection();
         this.saveUndoState();
+        this.setSelection( range );
+
+        // Wait until the drop occurs then clean it. Ideally we would only do this
+        // for HTML drops, but some browsers send HTML as text/plain.
+        var self = this;
+        setTimeout( function () {
+            try {
+                cleanTree( self._root );
+                addLinks( self._root, self._root, self );
```

Comment:
This handles content dropped from outside the editor, say from another browser window. We could skip this for content dragged within the editor.